src/app/external-cluster-wizard/steps/external-cluster/provider/aks/component.ts

```diff
@@ -70,6 +70,11 @@ enum Mode {
   User = 'User',
 }
 
+export enum VMSizeState {
+  Empty_OR_Ready = 'VM Size',
```

Comment:
It could be just `Ready` I think and then we can use same value in both empty and ready state.